src/contribute-to-tidb/report-an-issue.md

```diff
@@ -0,0 +1,48 @@
+
+The first step before filing an issue report is to see whether the problem has already been reported. You can [use the search bar to search existing issues](https://docs.github.com/en/github/administering-a-repository/finding-information-in-a-repository/using-search-to-filter-issues-and-pull-requests). This doesn't always work, and sometimes it's hard to know what to search for, so consider this extra credit. We won't mind if you accidentally file a duplicate report. Don't blame yourself if your issue is closed as duplicated. We highly recommend if you are not sure about anything of your issue report, you can turn to [internal.tidb.io](https://internals.tidb.io) for a wider audience and ask for discussion or help.
+
+## Filing an issue
+
+If the problem you’re reporting is not already in the issue tracker, you can [open a GitHub issue](https://docs.github.com/en/issues/tracking-your-work-with-issues/creating-an-issue) with your GitHub account. TiDB uses issue template for different kinds of issues. Issue templates are a bundle of questions to collect necessary information about the problem so it's easy for other contributors to participate. For example a bug report issue template consists of four questions:
+
+  - Minimal reproduce step.
+  - What did you expect to see?
+  - What did you see instead?
+  - What is your TiDB version?
+
+Answering these questions give the details about your problem so other contributors or TiDB users could pick up your issue more easily. 
+
+As previous section shows, duplicated issues should be reduced. To help others who encountered the problem find your issue, except for problem details answered in the issue template, a descriptive title which contains information that might be unique to it also helps. This can be the components your issue belongs to or database features used in your issue, the conditions that trigger the bug, or part of the error message if there is any. 
+
+## Making good issues
+
+Except for a good title and detailed issue message, you can also add suitable labels to your issue via [\label](https://prow.tidb.io/command-help?repo=pingcap%2Ftidb#type), especially which component the issue belongs to. Many committers and contributors only focus on certain subsystems of TiDB. Setting the appropriate component is important for getting their attention.
+
+If you are able to, you should take more considerations on your issue:
+
+  - Does the feature fit well into TiDB's architecture? Will it scale and keep TiDB flexible for the future, or will the feature restrict TiDB in the future?
+  - Is the feature a significant new addition (rather than an improvement to an existing part)? If yes, will the community commit to maintaining this feature?
+  - Does this feature align well with currently ongoing efforts?
+  - Does the feature produce added value for TiDB users or developers? Or does it introduce the risk of regression without adding relevant user or developer benefit? 
```

Comment:
```suggestion
* Does the feature fit well into TiDB's architecture? Will it scale and keep TiDB flexible for the future, or will the feature restrict TiDB in the future?
* Is the feature a significant new addition (rather than an improvement to an existing part)? If yes, will the community commit to maintaining this feature?
* Does this feature align well with currently ongoing efforts?
* Does the feature produce additional value for TiDB users or developers? Or does it introduce the risk of regression without adding relevant user or developer benefit?
```